Correct livestream description (#5558)
segmentio/segment-docs: src/connections/spec/video.md

| `total_length`   | Integer   | The total duration of the content/asset in seconds. This should exclude the duration of any ads included in the playback of this asset. For livestream playback, send `null`.   |
| `channel`        | String    | The channel in which the video content is playing, such as `espn` or `my blog`.   |
| `full_episode`   | Boolean   | `true` if content is a full episode and `false` otherwise.   |

> | `livestream`     | Boolean   | If the playback will be a livetream, send `true`, otherwise `false`.   |

Reviewer: ```suggestion
| `livestream`     | Boolean   | If the playback is a livetream, send `true`, otherwise `false`.   |
```
